src/platform/tools/telemetryGenerator.node.ts

```diff
@@ -280,7 +280,7 @@ function writeTelemetryEntry(entry: TelemetryEntry) {
             const link = `https://github.com/microsoft/vscode-jupyter/tree/main/${l.file}`;
             writeOutput(`\n[${l.file}](${link})`);
             writeOutput('```typescript');
-            writeOutput(l.code);
+            writeOutput(l.code.replace('\\r\\n', '\n'));
```

Comment:
I believe the differences in the telemetry lately has been because of some us being on Windows and others not.

Reply:
Good catch Rich. Neither github.com nor github.dev (monaco diff editor) can render the line break changes ;(

Reply:
Makes sense. I was looking at the markdown and wondering what I was missing...